modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorage.java

```diff
@@ -1015,6 +1117,149 @@ public void destroyData(WriteBatch writeBatch) throws RocksDBException {
         writeBatch.deleteRange(cf, partitionStartPrefix(), partitionEndPrefix());
     }
 
+    @Override
+    public @Nullable TableRowAndRowId pollForVacuum(HybridTimestamp lowWatermark) {
```

Comment:
What about rebalancing?

If we are in the process of rebalancing, should we return rows? I think we should throw an exception, like in `RocksDbMvPartitionStorage#read`.

Since we clean up the storage at the beginning/aborting of the rebalance, I think that the queue should also be cleaned up.